alot/account.py

```diff
@@ -198,20 +198,11 @@ class Account:
     def __init__(self, address=None, aliases=None, alias_regexp=None,
                  realname=None, gpg_key=None, signature=None,
                  signature_filename=None, signature_as_attachment=False,
-                 sent_box=None, sent_tags=None, draft_box=None,
-                 draft_tags=None, replied_tags=None, passed_tags=None,
+                 sent_box=None, sent_tags=[], draft_box=None,
+                 draft_tags=[], replied_tags=[], passed_tags=[],
                  abook=None, sign_by_default=False,
                  encrypt_by_default=u"none", encrypt_to_self=None,
                  case_sensitive_username=False, **_):
-        sent_tags = sent_tags or []
```

Comment:
In light of the comment just now I mean that this line could be 
```py
sent_tags = sent_tags or ["send"]
```
and for drafts the corresponding one.

Reply:
Wouldn't this overwrite the empty list as parameter?
I mean if a user sets `send_tags = ,` in the ocnfig, (indicating no tags, i.e., the empty list), then `sent_tags or ["send"] ` would default to the latter, which is what @sgelb wanted to change..

And yes, you interpreted my comment above correctly :)

Reply:
You are right, this would make it impossible to completely unset the list from the config... Then we have to code it differently.

Reply:
`send_tags = ["send"] if send_tags  is None` ...